src/palette/qml/MuseScore/Palette/PaletteCellPropertiesDialog.qml

```diff
@@ -80,11 +80,30 @@ StyledDialogView {
                 id: repeater
 
                 model: [
-                    { title: qsTrc("palette", "X"), value: propertiesModel.xOffset, incrementStep: 1,
-                        //: Abbreviation of "spatium"
-                        measureUnit: qsTrc("global", "sp") },
-                    { title: qsTrc("palette", "Y"), value: propertiesModel.yOffset, incrementStep: 1, measureUnit: qsTrc("global", "sp") },
-                    { title: qsTrc("palette", "Content scale"), value: propertiesModel.scaleFactor, incrementStep: 0.1 }
+                    //: "sp" = abbreviation of "spatium"
```

Comment:
@mathesoncalum This was a special comment for the translators, to explain how to translate the string "sp". It is read by the tool that extracts strings from the source code. It needs to be on the line immediately above the line containing the string in question. Could you please fix this?